ddl/table_test.go

```diff
@@ -67,6 +69,43 @@ func testTableInfo(c *C, d *ddl, name string, num int) *model.TableInfo {
 	return tblInfo
 }
 
+// testViewInfo creates a test view with num int columns.
+func testViewInfo(c *C, d *ddl, name string, num int) *model.TableInfo {
```

Comment:
Can we reuse `testTableInfo` function? there are some redundant code.

Reply:
I think we should keep it independent, both function didn't have too much redundant code.